storage/sqlitekv/sqlitekv.go

```diff
@@ -0,0 +1,167 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+//go:generate dbx.v1 golang -d sqlite3 sqlitekv.dbx .
+
+package sqlitekv
+
+import (
+	"context"
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/zeebo/errs"
+	"storj.io/storj/internal/migrate"
+
+	"storj.io/storj/storage"
+)
+
+var ErrSqlitekv = errs.Class("sqlitekv error")
+
+type SqliteKV struct {
+	DB *DB
+}
+
+// New opens a new SqliteKV database connection.
+func New(path string) (storage.KeyValueStore, error) {
+	db, err := Open("sqlite3", path)
+	if err != nil {
+		return nil, ErrSqlitekv.Wrap(err)
+	}
+	sqliteDB := &SqliteKV{
+		DB: db,
+	}
+	return sqliteDB, nil
+}
+
+// Get gets a value to store
+func (db SqliteKV) Get(key storage.Key) (storage.Value, error) {
+	// TODO: should we add contexts to the key-value store interface?
+	row, err := db.DB.Find_Item_Value_By_Key(context.Background(), Item_Key(key))
+	if err != nil {
+		return nil, ErrSqlitekv.Wrap(err)
+	}
+	return row.Value, nil
+}
+
+// Put adds a value to store
+func (db SqliteKV) Put(key storage.Key, value storage.Value) error {
+	// TODO: should we add contexts to the key-value store interface?
```

Comment:
Why not?

What I see on not adding it now is:
Normally any store should have the context because the most of them rely on I/O or network access, adding it later to the methods will be breaking changes and refactoring will have to be taken